core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1028,8 +1037,8 @@ public void copyFileToContainer(MountableFile mountableLocalFile, String contain
     @Override
     public void copyFileFromContainer(String containerPath, String destinationPath) throws IOException {
 
-        if (!isRunning()) {
-            throw new IllegalStateException("copyFileToContainer can only be used while the Container is running");
+        if (!isCreated()) {
```

Comment:
Kind of like above, `wasCreated()` seems more clear.
Would change the exception message accordingly to "after the container was created".

Reply:
this is a public (default) method from `ContainerState.java` which is only used twice in the project. Still I do not feel comfortable to rename it.

Reply:
Okay, let's not rename it if we break the public API.